src/Core/Addon/Module/ModuleManager.php

```diff
@@ -752,4 +775,17 @@ private function checkConfirmationGiven($action, Module $module)
             }
         }
     }
+
+    /**
+     * Clear smarty and Symfony cache (the sf2 cache is remove on the process shutdown).
+     */
+    private function clearCache()
```

Comment:
why not include prestashop cache_clearer service instead?

Reply:
yep better!